libraries/lib-effects/tests/CompressorProcessorTests.cpp

```diff
@@ -0,0 +1,47 @@
+/*  SPDX-License-Identifier: GPL-2.0-or-later */
+/*!********************************************************************
+
+  Audacity: A Digital Audio Editor
+
+  CompressorProcessorTests.cpp
+
+  Matthieu Hodgkinson
+
+**********************************************************************/
+#pragma once
+
+#include "processors/CompressorProcessor.h"
+#include <catch2/catch.hpp>
+
+TEST_CASE("CompressorProcessor", "smoke test")
```

Comment:
It's not quite clear what it tests TBH

Reply:
It doesn't test much indeed. I've been using it a little bit as debug helper. Maybe it will come in handy when I start tweaking the algorithm, which I consider doing after the UI follow-up work.